scripts/check-go-vulnerabilities.sh

```diff
@@ -0,0 +1,33 @@
+
+#!/usr/bin/env bash
+
+# This script checks for go vulnerabilities.
+
+govul="govulncheck"
+
+# Test if govulncheck is installed as a binary or a go module.
+if command -v govulncheck &> /dev/null; then
+    # If the user has installed govulncheck as a binary, use the binary.
+    govul=$(command -v govulncheck)
+elif [ -n "$GOPATH" ] && [ -f "$GOPATH/bin/govulncheck" ]; then
+    govul="$GOPATH/bin/govulncheck"
+elif [ -n "$GOBIN" ] && [ -f "$GOBIN/govulncheck" ]; then
+    govul="$GOBIN/govulncheck"
+fi
+
+# If govulncheck is not installed, exit with an error.
+if ! command -v "$govul" &> /dev/null; then
+    echo "govulncheck is not installed."
+    exit 1
+fi
+
+result=$($govul ./...)
+if [ $? -eq 0 ]; then
```

Comment:
It might be nice if this prints a message so that users know no vulnerabilities were found. This distinguishes it from the script crashing or exiting without a useful error message.

Reply:
I also changed it for the swaggo fmt file to be more consistent/helpful